source/_integrations/meteoclimatic.markdown

```diff
@@ -18,7 +20,30 @@ The Meteoclimatic integration uses the [Meteoclimatic](https://www.meteoclimatic
 There is currently support for the following platforms within Home Assistant:
 
 - Weather
+- [Sensor](#sensor-platforms)
 
-It displays the current weather reported by specific Meteoclimatic stations.
+It displays the current weather along with individual sensors that include daily maximum and minimum values.
 
 {% include integrations/config_flow.md %}
+
+## Sensor platforms
+
+All the following sensors will be created:
+
+|Entity|Description|
+|------|-----------|
+|`daily_max_humidity`|Maximum humidity for the past 24 hours in %|
```

Comment:
The entity naming is not guaranteed and can be customized by the user.
In general it is recommend to describe the sensors, instead of providing something that looks like an entity ID.